cli/src/res/btc_genesis_header_mainnet.json

```diff
@@ -1,11 +1,10 @@
 {
     "version": 536870912,
-    "previous_header_hash": "0000000000000000000a4adf6c5192128535d4dcb56cfb5753755f8d392b26bf",
-    "merkle_root_hash": "1d21e60acb0b12e5cfd3f775edb647f982a2d666f9886b2f61ea5e72577b0f5e",
+    "previous_header_hash": "0x0000000000000000000a4adf6c5192128535d4dcb56cfb5753755f8d392b26bf",
+    "merkle_root_hash": "0x1d21e60acb0b12e5cfd3f775edb647f982a2d666f9886b2f61ea5e72577b0f5e",
     "time": 1558168296,
     "bits": 388627269,
     "nonce": 1439505020,
     "height": 576576,
-    "hash": "0000000000000000001721f58deb88b0710295a02551f0dde1e2e231a15f1882",
-    "network_id": "Mainnet"
-}
\ No newline at end of file
+    "hash": "0x0000000000000000001721f58deb88b0710295a02551f0dde1e2e231a15f1882"
+}
```

Comment:
These two JSON files should not be altered directly, for they are generated by the script https://github.com/chainx-org/ChainX/tree/develop-2.0/scripts/genesis#btc_genesis_header_genpy. And I suggest not changing them as I don't see any benefits.